test/core/modules/image-sequencer.js

```diff
@@ -165,6 +165,22 @@ test('insertSteps({image: {index: index, name: "module", o: options} }) inserts
 });
 
 
+test('getSteps() returns correct array of steps', function(t){
+  var stepsOptionsName = ['load-image', 'channel', 'channel', 'channel', 'channel', 'channel'];
```

Comment:
This looks great! 

Why don't we do this with a new instance of ImageSequencer so that we can control the exact steps passed in, and it isn't interdependent on prior tests? What do you think?

And we can also add this to the README!

Reply:
Sure.
Thanks for the review.

Reply:
Done